axelrod/tests/unit/test_resultset.py

```diff
@@ -190,6 +190,52 @@ def test_init(self):
         self.assertEqual(rs.players, self.players)
         self.assertEqual(rs.num_players, len(self.players))
 
+    def _clear_matrix(self, matrix):
+        for i in range(len(matrix)):
```

Comment:
FYI, this is sometimes considered an anti pattern and the following is preferred:

```
for i, row in enumerate(matrix):
    for j, _ in enumerate(row):
         matrix[i, j] = 0
```